### PR TITLE
fix: Remove error log message when logging set to STDOUT

### PR DESCRIPTION
**Pull request by lenny-intel**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/go-mod-core-contracts/blob/master/.github/Contributing.md.

## What is the current behavior?
Bogus error log message logged when logging configured for STDOUT

## What is the new behavior?
No error message logged for this logging configuration.
Need for Security Utility to run in silent mode

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## New Imports

- [ ] Yes
- [x] No

## Specific Instructions
Are there any specific instructions or things that should be known prior to reviewing?

## Other information

### CODE REVIEW

File: clients/logger/logger.go [hunk 1]
```diff
@@ -69,11 +69,6 @@ type fileWriter struct {
 // NewClient creates an instance of LoggingClient
 func NewClient(owningServiceName string, isRemote bool, logTarget string, logLevel string) LoggingClient {
 	lc := newClient(owningServiceName, isRemote, logTarget, logLevel)
-
-	if logTarget == "" {
-		lc.Error("logTarget cannot be blank, using stdout only")
```
Comment (jim-wang-intel) [inline clients/logger/logger.go:74]: instead of removing it, should we just change the logging level to debug?

**Review by jim-wang-intel — COMMENTED** [review]
LGTM

File: clients/logger/logger.go (hunk 1 above)
Comment (lenny-intel) [reply clients/logger/logger.go:74]: Nope, this is a valid configuration and will soon be the only configuration.

Comment (jim-wang-intel) [reply clients/logger/logger.go:74]: got it, good to know!

**Review by bnevis-i — APPROVED** [review]

**Comment by lenny-intel** [comment]
recheck

**Review by cloudxxx8 — APPROVED** [review]
LGTM

**Pull request merged**